haproxy_config/haproxy_config.py

```diff
@@ -73,16 +97,30 @@ def get_config():
       port = environment.get('VIRTUAL_PORT')
     if not port:
         port = 80
+        
+    http_auth_user = environment.get("HTTP_AUTH_USER") or ''
+    http_auth_pw = environment.get("HTTP_AUTH_PASS") or ''
+    http_auth_pw_type = environment.get("HTTP_AUTH_SECURE_PASSWORD") or False
+        
+    basic_auth = {}
+    
+    if http_auth_pw and http_auth_pw :
+      basic_auth['user'] =  http_auth_user
+      basic_auth['password'] =  http_auth_user
```

Comment:
This also looks wrong to me, and seems broken on the test-instance. Shouldnt that be using the password instead of the user name?